GIFrameworkMaps.Web/Scripts/Panels/PrintPanel.ts

```diff
@@ -82,14 +83,34 @@ export class PrintPanel implements SidebarPanel {
   cancelledByUser: boolean = false;
   longLoadingTimeout: number;
   longLoadingTimeoutLength: number = 10000;
+  printConfiguration: PrintConfiguration;
 
   constructor(container: string) {
     this.container = container;
   }
-  init() {
+  async init() {
+    const resp = await fetch(`${document.location.protocol}//${this.gifwMapInstance.config.appRoot}print/configuration/${this.gifwMapInstance.config.id}`);
```

Comment:
For organisation sake, could this new block of code be moved into a separate function. Something like 
```
private async getPrintConfiguration(){
   //this block of fetching and error handling code
}
```

Usage:
`this.printConfiguration = await this.getPrintConfiguration();`